context/app/static/js/pages/Template/Template.tsx

```diff
@@ -0,0 +1,163 @@
+                        key={name}
+                      >
+                        {idx === assay_display_name.length - 1 ? name : `${name}, `}
+                      </InternalLink>
+                    ))}
+                  </Stack>
+                </LabelledSectionText>
+              )}
+            </Stack>
+            <WorkspaceDatasetsTable datasetsUUIDs={[...datasets]} isSelectable={false} />
+          </Stack>
+        </AccordionDetails>
+      </StyledAccordion>
+      <NewWorkspaceDialogFromExample example={example} {...rest} />
+    </>
+  );
+}
+
+interface TemplatePageProps {
+  templateKey: string;
+}
+
+function Template({ templateKey }: TemplatePageProps) {
+  const { templates } = useWorkspaceTemplates();
+  const template = templates[templateKey];
+  const datasetTypeMap = useDatasetTypeMap();
+
+  if (!template) {
+    return null;
+  }
+
+  return (
+    <Stack spacing={4} marginBottom={5}>
+      <Stack spacing={2}>
+        <SummaryData
+          title={template.title}
+          entity_type="WorkspaceTemplate"
+          entityTypeDisplay="Workspace Template"
+          status=""
+          mapped_data_access_level=""
+        />
+        <Stack component={SummaryPaper} spacing={1}>
+          <LabelledSectionText label="Description">{template.description}</LabelledSectionText>
+          <LabelledSectionText label="Tags">
+            <Stack spacing={1} marginTop={1} direction="row">
+              {template.tags.map((tag) => (
+                <Chip
+                  key={tag}
+                  label={tag}
+                  variant="outlined"
+                  sx={(theme) => ({ borderRadius: '.5rem', borderColor: theme.palette.grey[200] })}
+                />
+              ))}
+            </Stack>
+          </LabelledSectionText>
+        </Stack>
+      </Stack>
+      {template.examples && (
+        <Stack spacing={1}>
+          <Typography variant="h4">Sample Workspaces</Typography>
+          {isAuthenticated ? (
+            <IconPanel status="info">
+              Sample workspaces are provided to help you get started with this template and to better understand the
+              types of data that are compatible with it.
+            </IconPanel>
+          ) : (
+            <LogInPanel>
+              Sample workspaces are available to help you get started with this template and better understand the types
+              of compatible data. Please <InternalLink href="/login">log in</InternalLink> to explore a sample
+              workspace.
+            </LogInPanel>
+          )}
+          {template.examples.map((example, idx) => (
+            <ExampleAccordion
+              key={example.title}
+              example={example}
+              templateKey={templateKey}
+              defaultExpanded={idx === 0}
+              datasetTypeMap={datasetTypeMap}
```

Comment:
SWR caches the results and should only make a single network request for a given key. We can move the call of `useDatasetTypeMap` directly into `ExampleAccordion`.

Reply:
Ah nice!